packages/packages/locations/src/injections.tsx

```diff
@@ -0,0 +1,63 @@
+import { Location, Filler, Injection, InjectionOptions, Name } from './types';
+import FillerWrapper from './components/filler-wrapper';
+
+const DEFAULT_PRIORITY = 10;
+
+const injections: Map<string, Injection> = new Map();
```

Comment:
```suggestion
const injections: Map<InjectionId, Injection> = new Map();
```

Maybe? just for readability

Reply:
Done